.github/workflows/buf-push.yml

```diff
@@ -0,0 +1,22 @@
+name: Buf (push)
+on:
+  push:
+    branches:
+    - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: bufbuild/buf-setup-action@v1
+    - uses: bufbuild/buf-lint-action@v1
+      with:
+        input: proto
+    # - uses: bufbuild/buf-breaking-action@v1
+    #   with:
+    #     input: proto
+    #     against: 'https://github.com/${GITHUB_REPOSITORY}.git#branch=main'
+    - uses: bufbuild/buf-push-action@v1
```

Comment:
Nice! Thanks =)

Reply:
Discussing this with Steve, we've found that the repetition (managing buf-push and buf-pull-request separately) may be more explicit and safer since we really don't want to accidentally push to the BSR.